source/custom/custom.go

```diff
@@ -46,5 +49,64 @@ func (s Source) Name() string { return "custom" }
 // Discover features
 func (s Source) Discover() (source.Features, error) {
 	features := source.Features{}
+	allFeatureConfig := s.mergeFeatures(getStaticFeatureConfig(), Config)
```

Comment:
Do we need to merge? Would just a simple append work? First run the static and the user-defined ones. It might even be desirable to have multiple rules for the same label. WDYT?

Reply:
I have no special Preference, We can do this as you suggest.
The current logic allows a user to override a specific feature that was defined statically.

Reply:
@zvonkok do you have any preference? I think, later on, we could add a config option to disable the static rule(s) if there is need for that. WDYT?

Reply:
Let's go on and do append like how it is now in the code. Implicit behaviour is sometimes misleading and can lead to error. I am in favour of explicitly disabling the static options via config.